add-graphics-renderer/src/main/java/com/esri/arcgisruntime/sample/addgraphicsrenderer/MainActivity.java

```diff
@@ -46,65 +45,71 @@ protected void onCreate(Bundle savedInstanceState) {
+        // yellow diamond point symbol
+        SimpleMarkerSymbol pointSymbol = new SimpleMarkerSymbol(Color.YELLOW, 10, SimpleMarkerSymbol.Style.DIAMOND);
+        // create graphic for point
+        Graphic pointGraphic = new Graphic(pointGeometry);
+        // create a graphic overlay for the point
+        GraphicsOverlay pointGraphicOverlay = new GraphicsOverlay();
         // create simple renderer
-        SimpleRenderer simpleRenderer = new SimpleRenderer();
-        // create point symbol
-        SimpleMarkerSymbol pointSymbol = new SimpleMarkerSymbol(yellow, 30, SimpleMarkerSymbol.Style.SQUARE);
-        // set symbol to renderer
-        simpleRenderer.setSymbol(pointSymbol);
-        // set renderer to graphics overlay
-        grOverlay.setRenderer(simpleRenderer);
-
+        SimpleRenderer pointRenderer = new SimpleRenderer(pointSymbol);
+        pointGraphicOverlay.setRenderer(pointRenderer);
+        // add graphic to overlay
+        pointGraphicOverlay.getGraphics().add(pointGraphic);
+        // add graphics overlay to the MapView
+        mMapView.getGraphicsOverlays().add(pointGraphicOverlay);
+
+        // line graphic
+        PolylineBuilder lineGeometry = new PolylineBuilder(SpatialReferences.getWebMercator());
+        lineGeometry.addPoint(-10e5, 40e5);
+        lineGeometry.addPoint(20e5, 50e5);
+        // solid blue line symbol
+        SimpleLineSymbol lineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, Color.BLUE, 5);
+        // create graphic for polyline
+        Graphic lineGraphic = new Graphic(lineGeometry.toGeometry());
+        // create graphic overlay for polyline
+        GraphicsOverlay lineGraphicOverlay = new GraphicsOverlay();
+        // create simple renderer
+        SimpleRenderer lineRenderer = new SimpleRenderer(lineSymbol);
+        // add graphic to overlay
+        lineGraphicOverlay.setRenderer(lineRenderer);
+        // add graphic to overlay
+        lineGraphicOverlay.getGraphics().add(lineGraphic);
         // add graphics overlay to the MapView
-        mMapView.getGraphicsOverlays().add(grOverlay);
+        mMapView.getGraphicsOverlays().add(lineGraphicOverlay);
+
+        //polygon graphic
+        PolygonBuilder polygonGeometry = new PolygonBuilder(SpatialReferences.getWebMercator());
+        polygonGeometry.addPoint(-20e5, 20e5);
+        polygonGeometry.addPoint(20e5, 20e5);
+        polygonGeometry.addPoint(20e5, -20e5);
+        polygonGeometry.addPoint(-20e5, -20e5);
+        // solid yellow polygon symbol
+        SimpleFillSymbol polygonSymbol = new SimpleFillSymbol(Color.YELLOW, SimpleFillSymbol.Style.SOLID, null);
+        // create graphic for polygon
+        Graphic polygonGraphic = new Graphic(polygonGeometry.toGeometry());
+        // create graphic overlay for polygon
+        GraphicsOverlay polygonGraphicOverlay = new GraphicsOverlay();
+        // create simple renderer
+        SimpleRenderer polygonRenderer = new SimpleRenderer(polygonSymbol);
```

Comment:
Structure-wise, this sample is all about creating renderer and setting it on a graphics layer, so would be nice if this code was somewhat earlier on, instead of having to scroll all the way to the bottom to find the pertinent lines.